src/Microsoft.IdentityModel.Protocols.SignedHttpRequest/GlobalSuppressions.cs

```diff
@@ -11,3 +11,4 @@
 [assembly: SuppressMessage("Performance", "CA1825: Avoid zero-length array allocations", Justification = "net45 target doesn't support Array.Empty")]
 [assembly: SuppressMessage("Globalization", "CA1308:Normalize strings to uppercase", Justification = "Headers need to be lowercase to calcuate appropriate hash", Scope = "type", Target = "~T:Microsoft.IdentityModel.Protocols.SignedHttpRequest.SignedHttpRequestHandler")]
 [assembly: SuppressMessage("Design", "CA1001:Types that own disposable fields should be disposable", Justification = "Breaking change", Scope = "type", Target = "~T:Microsoft.IdentityModel.Protocols.SignedHttpRequest.SignedHttpRequestHandler")]
+[assembly: SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "<Pending>", Scope = "member", Target = "~P:Microsoft.IdentityModel.Protocols.SignedHttpRequest.SignedHttpRequestInvalidNonceClaimException.PropertyBag")]
```

Comment:
Change the justification to be "Want to be consistent with other PropertyBag" properties in the library

Reply:
@GeoK @mafurman @RojaEnnam - Any objections with this? I think in the perfect world we'd be able to follow this across the library but since all the other property bags are not readonly, it would be pretty confusing to have this one be a good citizen.

Reply:
As discussed, agree with being consistent with other PropertyBag.